galaxy/importer/loaders/apb.py

```diff
@@ -137,6 +149,13 @@ def _check_plans(self):
                         expected_parameter_keys,
                         'plans[{0}].parameters[{1}]'.format(idx, pidx),
                         params)
+                    for param_key in params.keys():
```

Comment:
I really had mixed feelings about adding this logic here as it's really doing validation of plans. My reasoning for including it here is two-fold: 1) easiest 2) most of the work to get down to the plans + parameters has already been done.